### DISCUSSION TITLE
Zustand Persist + Redis

### DISCUSSION THREAD

**Discussion by rizkiandrianto** [pmndrs/zustand#1724] [Q&A]
Based on [this comment](https://github.com/pmndrs/zustand/issues/510#issuecomment-887431215), I want to know if there is possibility to zustand to work with persist in redis?
Maybe either via another API call, or direct save?

**Comment by JacobWeisenburger** [comment]
feel free to use this template:
[Zustand custom persist storage template](https://gist.github.com/JacobWeisenburger/eabdb5cadc947e9d35f45d6b7cf2710e)

If you found my answer satisfactory, please consider supporting me. Even a small amount is greatly appreciated. Thanks friend! 🙏
https://github.com/sponsors/JacobWeisenburger
